src/core_modules/capture-core/components/CardList/CardList.selectors.js

```diff
@@ -1,25 +1,24 @@
 // @flow
 import { createSelector } from 'reselect';
-import { chunk } from 'capture-core-utils';
 import { elementTypes } from '../../metaData/DataElement';
 
 
-const elementsSelector = props => props.dataElements;
+const elementsSelector = dataElements => dataElements;
 
 // $FlowFixMe[missing-annot] automated comment
-const makeElementsContainerSelector = () => createSelector(
+export const makeElementsContainerSelector = () => createSelector(
     elementsSelector,
     (elements) => {
-        const newElements = [...elements];
         // $FlowFixMe[prop-missing] automated comment
         const imageDataElement = elements.find(a => a.type === elementTypes.IMAGE);
```

Comment:
I can see that currently the type is not part of the elements array and when adding a tracked entity attribute of image value type the element is not part of this array at all. Great if you could look into this.

Reply:
@JoakimSM excellent point. I have addressed it I believe in my last commit